source/Interface.cpp

```diff
@@ -122,20 +124,20 @@ void Interface::Load(const DataNode &node)
 		else
 		{
 			// Check if this node specifies a known element type.
-			if(child.Token(0) == "sprite" || child.Token(0) == "image" || child.Token(0) == "outline")
+			if(key == "sprite" || key == "image" || key == "outline")
 				elements.push_back(new ImageElement(child, anchor));
-			else if(child.Token(0) == "label" || child.Token(0) == "string" || child.Token(0) == "button"
-					|| child.Token(0) == "dynamic button")
+			else if(key == "label" || key == "string" || key == "button"
+					|| key == "dynamic button")
```

Comment:
Looks like it fits within the line size limit now.
```suggestion
			else if(key == "label" || key == "string" || key == "button" || key == "dynamic button")
```